config/webpack.common.js

```diff
@@ -83,8 +75,11 @@ module.exports = (prodMode) =>
                     }
                 }
             }),
+            new AssetsWebpackPlugin({
+                filename: 'assets.json',
+                path: path.resolve('./dist/server')
+            }),
```

Comment:
Could you paste in what the output looks like in a comment (just here on the code review, so I can try to imagine what it would work like)?  Thanks!